parlai/core/worlds.py

```diff
@@ -1634,3 +1634,16 @@ def create_task(opt: Opt, user_agents, default_world=None):
         world = BatchWorld(opt, world)
 
     return world
+
+
+def create_task_from_shared(shared_world):
```

Comment:
we don't already have something like this?

Reply:
Semi-surprisingly, no. We just do something like `shared['world_class'](opt, shared=shared)` normally.

Reply:
I’m wondering world.clone() isn’t better. We have this for Agent but nothing else, and I kinda like clones better